packages/common/src/components/types.ts

```diff
@@ -11,6 +11,7 @@ export interface Field {
   toLabel(t: (key: string) => string): string;
   isVisible?: boolean;
   isIdentity?: boolean;
+  isAction?: boolean;
```

Comment:
can we make it `isManageableColumn` ?

because soon we  will have a column for checkboxs (for multi column actions) and a toggel for expanding a raw ... all this columns will not be manageable columns, not sortable, not searchable and not showing on manage column modal

Reply:
@yaacov 
I've revisited the problem and it's not that straightforward to rename:
1. `isManagableColumn` would be misleading as the current implementation is action-specific. The columns with `isAction` flag are put at the end of the column list. We could provide a generic implementation for a non-managable column on any postion (i.e. index=0,1,2, ...) but this would require more testing (and we don't have a use case yet).
2. it's easier to group low level flags when neededi.e. `isManagableColumn === !isAction && !isSelect` than the other way around - "split" high level flag like `isManagableColumn`.  Sooner or later you need a condition like `if(column.isAction){ // do sth action-specific}`

Reply:
ehh, ok :-)